<launch>
    <node
        pkg="coral_spawn_imager"
        type="simple_publisher.py"
        name="pub"
        output="screen"
    />

    <param name="camera_index" type="int" value="1"/>
    <param name="image_width" type="int" value="1920" />
    <param name="image_height" type="int" value="1080" />
    <param name="iso" type="int" value="400" />
    <param name="red_gain" type="double" value="-1.0" />
    <param name="blue_gain" type="double" value="-1.0" />
    <param name="awb_mode" type="string" value="auto" />
    <param name="shutter_speed" type="int" value="10000" />
    <paran name="exposure_mode" type="string" value="auto" />
</launch>
